### PR TITLE
Thredds data target

### LINKED ISSUE #26: Thredds data targets
**Is your feature request related to a problem? Please describe.**
As it stands, `sandpiper`'s docker instance needs to have storage mounted in order to perform the data collection and processing. This requires the user to have direct access to the data which goes against the idea of DACCS.

**Describe the solution you'd like**
`p2a_impacts` will likely need some drastic alterations to target `thredds urls` rather than direct files. For now, the impact to this repo should just be a version update to the parent package.

**Describe alternatives you've considered**
Since `thredds` is our main data target I don't believe there is another good alternative at the moment.

### PR DESCRIPTION
**Pull request by nikola-rados**
A demo container is running on `30810` which you can use to test the process.

Resolves #26.

### CODE REVIEW

File: sandpiper/processes/wps_resolve_rules.py [hunk 1]
```diff
@@ -73,6 +73,15 @@ def __init__(self):
                 default="p2a_rules",
                 data_type="string",
             ),
+            LiteralInput(
+                "thredds",
+                "Thredds",
+                abstract="Data from thredds server. It is not recommended to change from the default (True)",
```
Comment (sum1lim) [inline sandpiper/processes/wps_resolve_rules.py:79]: I guess this is why you did not add tests with the input in the demo?

**Review by sum1lim — APPROVED** [review]
Looking good. Nice work!

File: sandpiper/processes/wps_resolve_rules.py (hunk 1 above)
Comment (nikola-rados) [reply sandpiper/processes/wps_resolve_rules.py:79]: Well actually the test runs with the `False` flag. I didn't add a case for true, I can do that now.

**Review by cairosanders — APPROVED** [review]
Sorry I didn't notice this earlier, works for me!

**Pull request merged**